contrib/!source-control/github/packages.el

```diff
@@ -86,3 +88,29 @@
         "gfC" 'spacemacs/git-link-commit-copy-url-only)
       ;; default is to open the generated link
       (setq git-link-open-in-browser t))))
+
+(defun github/init-magit-gh-pulls ())
+
+(defun github/pre-init-magit ()
```

Comment:
we should document why we do it in a use-package :pre-config hook for `magit`, this is not conventional stuff.